<?xml version="1.0"?>
<launch>
	
	<!-- equirectangular_convertor -->
	<include file="$(find place_recognition)/launch/utils/equirectangular_convertor.launch" />

	<!-- keypoints_detector -->
	<node pkg="place_recognition" type="keypoints_detector" name="keypoints_detector" output="screen" >
		<!-- remap -->
		<remap from="img_in" to="roomba1/equirectangular/image_raw" />
	</node>

</launch>